test/asn1_time_test.c

```diff
@@ -0,0 +1,275 @@
+            TEST_info("ASN1_TIME_diff(day=%d, sec=%d, %s) after ASN1_TIME_set_string_gmt() failed", day, sec, td->data);
+            local_error = error = 1;
+        }
+        if (!TEST_int_eq(ASN1_TIME_cmp_time_t(ptime, gtime_t), td->cmp_result)) {
+            TEST_info("ASN1_TIME_cmp_time_t(%s) after ASN1_TIME_set_string_gnt() to baseline bad comparison", td->data);
+            local_error = error = 1;
+        }
+        if (local_error)
+            TEST_info("ASN1_TIME_set_string_gmt() = %*s", ptime->length, ptime->data);
+        ASN1_TIME_free(ptime);
+    }
+
+    ptime = ASN1_TIME_new();
+    if (!TEST_ptr(ptime)) {
+        TEST_info("ASN1_TIME_new() failed");
+        error = 1;
+    } else {
+        int local_error = 0;
+        if (!TEST_int_eq(ASN1_TIME_set_string(ptime, td->data), td->check_result)) {
+            TEST_info("ASN1_TIME_set_string(%s) failed", td->data);
+            local_error = error = 1;
+        }
+        day = sec = 0;
+        if (!TEST_true(ASN1_TIME_diff(&day, &sec, ptime, &atime)) || !TEST_int_eq(day, 0) || !TEST_int_eq(sec, 0)) {
+            TEST_info("ASN1_TIME_diff(day=%d, sec=%d, %s) after ASN1_TIME_set_string() failed", day, sec, td->data);
+            local_error = error = 1;
+        }
+        if (!TEST_int_eq(ASN1_TIME_cmp_time_t(ptime, gtime_t), td->cmp_result)) {
+            TEST_info("ASN1_TIME_cmp_time_t(%s) after ASN1_TIME_set_string() to baseline bad comparison", td->data);
+            local_error = error = 1;
+        }
+        if (local_error)
+            TEST_info("ASN1_TIME_set_string() = %*s", ptime->length, ptime->data);
+        ASN1_TIME_free(ptime);
+    }
+
+    if (td->type == V_ASN1_UTCTIME) {
+        ptime = ASN1_TIME_to_generalizedtime(&atime, NULL);
+        if (td->convert_result == 1 && !TEST_ptr(ptime)) {
+            TEST_info("ASN1_TIME_to_generalizedtime(%s) failed", atime.data);
+            error = 1;
+        } else if (td->convert_result == 0 && !TEST_ptr_null(ptime)) {
+            TEST_info("ASN1_TIME_to_generalizedtime(%s) should have failed", atime.data);
+            error = 1;
+        }
+        if (ptime != NULL && !TEST_int_eq(ASN1_TIME_cmp_time_t(ptime, td->t), 0)) {
+            TEST_info("ASN1_TIME_to_generalizedtime(%s->%s) bad result", atime.data, ptime->data);
+            error = 1;
+        }
+        ASN1_TIME_free(ptime);
+    }
+    /* else cannot simply convert GENERALIZEDTIME to UTCTIME */
+
+    if (error)
+        TEST_error("atime=%s", atime.data);
+
+    return !error;
+}
+
+static int test_table_pos(int idx)
+{
+    return test_table(tbl_testdata_pos, idx);
+}
+
+static int test_table_neg(int idx)
+{
+    return test_table(tbl_testdata_neg, idx);
+}
+
+static int test_table_pos_64bit(int idx)
+{
+    return test_table(tbl_testdata_pos_64bit, idx);
+}
+
+static int test_table_neg_64bit(int idx)
+{
+    return test_table(tbl_testdata_neg_64bit, idx);
+}
+
+void register_tests(void)
```

Comment:
The tests have changed in master.  Instead use:

```
int setup_tests(void)
```

and add a `return 1;` to the end of the function.  Worth a rebase to head.